resources/state_of_the_dapps.json

```diff
@@ -0,0 +1,43 @@
+{
+    "auction-house":
+    {
+        "name":
+        {
+            "en": "Auction House"
+        },
+        "photo-path": "contacts://auction-house",
+        "state-of-the-dapp?": true,
```

Comment:
I could add `state-of-the-dapp?` from oustide when reading the json.
Is that what you mean?